docs/getting-started/handbooks/console/endpoints.mdx

```diff
@@ -0,0 +1,121 @@
+---
+id: endpoints
+title: Endpoints
+sidebar_label: Endpoints
+---
+
+In this guide, we will see how to configure the various types of endpoints available in my platform console, including:
+- [CRUD](/getting-started/handbooks/console/endpoints.mdx#create-a-crud-endpoint)
```

Comment:
```suggestion
- [CRUD](#create-a-crud-endpoint)
```
You can only use the anchor for page links.

Reply:
Done